




























assertThrows("f()", ReferenceError);
assertThrows("%f()", Error);
assertThrows("%_f()", SyntaxError);
